### PR TITLE
Created issue template

### PR DESCRIPTION
**Pull request by katowulf**

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+IMPORTANT! YOU MUST FOLLOW THESE INSTRUCTIONS OR YOUR ISSUE WILL BE CLOSED.
+
+Thank you for contributing to the Firebase community!
+
+Have a usage question about Functions?
+======================================
+We get lots of those and we love helping you, but the samples issue tracker is not the best place for them and they
+will be closed. Here are some resources to get help:
+
+- Go through the Developer's Guide: https://github.com/angular/angularfire2#developer-guide
```
Comment (jamesdaniels) [inline ISSUE_TEMPLATE.md]: fix link

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- Read the full API reference: https://github.com/angular/angularfire2/blob/master/docs/api-reference.md
```
Comment (jamesdaniels) [inline ISSUE_TEMPLATE.md]: ditto

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+**Failing Function code used (including require/import commands at the top)
+
+**Steps to set up and reproduce**
+
+<!-- detailed instructions to run your minimal repro or to recreate the environment -->
+
+**Sample data pasted or attached as JSON (not an image)**
+
+<-- paste/attach/link sample data as json (not an image) -->
```
Comment (jamesdaniels) [inline ISSUE_TEMPLATE.md]: `<!--` ?

File: ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+**Security rules used**
+
+<-- paste/attach/link security rules as json (not an image) -->
```
Comment (jamesdaniels) [inline ISSUE_TEMPLATE.md]: `<!--` ?

File: ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+We get lots of those and we love helping you, but the samples issue tracker is not the best place for them and they
+will be closed. Here are some resources to get help:
+
+- Go through the Developer's Guide: https://github.com/angular/angularfire2#developer-guide
+- Read the full API reference: https://github.com/angular/angularfire2/blob/master/docs/api-reference.md
+
+If the official documentation doesn't help, try asking through our officially supported channels:
+
+- Firebase Google Group: https://groups.google.com/forum/#!forum/firebase-talk
+- Stack Overflow: https://stackoverflow.com/questions/tagged/cloud-functions (include the firebase tag, too!)
```
Comment (jamesdaniels) [inline ISSUE_TEMPLATE.md]: link should be https://stackoverflow.com/questions/tagged/google-cloud-functions+firebase can then nuke the `(include the firebase tag, too!)`

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (katowulf) [inline ISSUE_TEMPLATE.md]: Ugh. 😞 😞 😞

File: ISSUE_TEMPLATE.md (hunk 5 above)
Comment (katowulf) [inline ISSUE_TEMPLATE.md]: Wow. Never knew about this. Should update other templates as well.

**Comment by katowulf** [comment]
Thanks, @jamesdaniels! All comments addressed.

**Pull request merged**